packages/scaffold-ui/src/partials/w3m-payment-token-button/index.ts

```diff
@@ -0,0 +1,30 @@
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+import { property } from 'lit/decorators.js'
+import { ifDefined } from 'lit/directives/if-defined.js'
+
+@customElement('w3m-payment-token-button')
+export class W3mPaymentTokenButton extends LitElement {
+  // -- State & Properties -------------------------------- //
+  @property() public tokenCurrency = ''
+
+  @property() public tokenImageUrl = ''
```

Comment:
Since the component explains itself well with it's name, I think we can be less repetitive for the property names and remove the `token*` prefix IMO. 

```suggestion
  @property() public currency = ''

  @property() public imageUrl = ''
```